docs/cpp/raw-pointers.md

```diff
@@ -156,7 +156,11 @@ int main()
 }
 ```
 
-Certain arithmetic operations can be used on non-`const` pointers to make them point to another memory location. Pointers are incremented and decremented using the **`++`**, **`+=`**, **`-=`** and **`--`** operators. This technique can be used in arrays and is especially useful in buffers of untyped data. A `void*` gets incremented by the size of a **`char`** (1 byte). A typed pointer gets incremented by size of the type it points to.
```

Comment:
> A `void*` gets incremented by the size of a **`char`** (1 byte).

This implies that pointer arithmetic can be done on `void*`, but that causes a hard error, since it is non-standard and only available as an extension on GCC (not sure if this extension exists for MSVC). Hence, this sentence is removed, but we can consider explicitly stating that it is not allowed.